src/promptflow/promptflow/_sdk/entities/_chat_group/_chat_group.py

```diff
@@ -0,0 +1,208 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import time
+from itertools import cycle
+from typing import Any, Dict, List, Optional
+
+from promptflow._sdk._constants import STOP_SIGNAL, ChatGroupSpeakOrder
+from promptflow._sdk._errors import ChatGroupError
+from promptflow._sdk.entities._chat_group._chat_role import ChatRole
+from promptflow._utils.logger_utils import get_cli_sdk_logger
+
+logger = get_cli_sdk_logger()
+
+
+class ChatGroup:
```

Comment:
An entity should have schema class?

Reply:
I'm planning to add schema and cli experiment in the next pr